visivo/commands/dbt_phase.py

```diff
@@ -0,0 +1,143 @@
+def _generate_sources(profiles, dbt_target):
+    from visivo.logging.logger import Logger
+    import click
+    from visivo.commands.utils import get_source_types
+
+    source_types = get_source_types()
+    sources = []
+    target_found = False
+    for profile_name, profile_value in profiles.items():
+        for target_name, target_value in profile_value["outputs"].items():
+            if target_name == dbt_target:
+                target_found = True
+            source = target_value
+            source["name"] = f"dbt_{profile_name}_{target_name}"
+            if "schema" in source:
+                source["db_schema"] = source.pop("schema")
+            if "user" in source:
+                source["username"] = source.pop("user")
+            if "type" in source and source["type"] == "snowflake":
+                Logger.instance().info(
+                    f"Configuring threads for snowflake is not supported.  Ignoring."
+                )
+                if "threads" in source:
+                    source.pop("threads")
+            if "type" in source and source["type"] not in source_types:
+                raise click.ClickException(
+                    f"Target type '{source['type']}' is not supported.  Only {source_types} are supported."
+                )
+            sources.append(source)
+
+    if not target_found:
+        raise click.ClickException(f"Target '{dbt_target}' not found in profiles.yml")
+
+    return sources
+
+
+def _generate_models(manifest, dbt_profile, dbt_target):
+    models = []
+    if "nodes" in manifest:
+        for path, node in manifest["nodes"].items():
+            if path.startswith("model."):
+                names = []
+                if "database" in node:
+                    names.append(node["database"])
+                if "schema" in node:
+                    names.append(node["schema"])
+                names.append(node["name"])
+                model = {
+                    "name": f"dbt_{node['name']}",
```

Comment:
I think we should give the user the ability to set the prefix `dbt_` in the dbt block. That way we have clear documentation how this behavior and people can just set it to blank if they dont want a prefix. 

Maybe something like this? 
```
dbt: 
  enabled: true
  prefix: "dbt_" #The prefix that visivo will use to create the names of your models and sources in the visivo dag
```